File: qc-core/src/main/java/com/eager/questioncloud/core/domain/point/service/UserPointService.java
package com.eager.questioncloud.core.domain.point.service;

import com.eager.questioncloud.core.domain.point.implement.UserPointReader;
import com.eager.questioncloud.core.domain.point.model.UserPoint;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

@Service
@RequiredArgsConstructor
public class UserPointService {
    private final UserPointReader userPointReader;

    public UserPoint getUserPoint(Long userId) {
        return userPointReader.getUserPoint(userId);
    }
}

File: qc-core/qc-point/qc-point-api/src/main/kotlin/com/eager/questioncloud/point/service/UserPointService.kt
package com.eager.questioncloud.point.service

import com.eager.questioncloud.point.domain.UserPoint
import com.eager.questioncloud.point.implement.UserPointReader
import org.springframework.stereotype.Service

@Service
class UserPointService(
    private val userPointReader: UserPointReader
) {
    fun getUserPoint(userId: Long): UserPoint {
        return userPointReader.getUserPoint(userId)
    }
}
